libraries/javascript/eddystone-advertising/README.md

```diff
@@ -0,0 +1,240 @@
+# Eddystone Advertising Library
+
+The Eddystone Advertising Library abstracts low level concepts of the Eddystone
+protocol and wraps existing Advertising APIs so that developers don't have to
+worry about these low level details. The library exposes simple functions that
+developers can use to advertise a Valid Eddystone packet from their device.
+
+** NOTE ** Currently only ChromeOS is supported.
+
+## Usage
+The Eddystone Advertising Library creates `window.eddystone` of type
+[`Eddystone`](#eddystone)
+
+To advertise a url:
+**Example**
+```js
+eddystone.registerAdvertisement({
+  type: 'url',
+  url: 'https://example.com',
+  txPower: -20
```

Comment:
FWIW, I like `advertisedTxPower`. It indicates that the value has no enforced relationship with reality. :)